googlecloud-plugins/docs/BigQuery-batchsource.md

```diff
@@ -0,0 +1,69 @@
+# Google BigQuery Batch Source
+
+
+Description
+-----------
+Batch source to use Google BigQuery as a Source.
+[link to BigQuery Docs](https://cloud.google.com/bigquery/docs/)
```

Comment:
Batch source to use Google Cloud Platform's [BigQuery](https://cloud.google.com/bigquery/docs/) as a source.